korman/operators/op_mesh.py

```diff
@@ -42,6 +49,10 @@ class PlasmaAddFlareOperator(PlasmaMeshOperator, bpy.types.Operator):
                                              description="Flare's distance from the illuminating object",
                                              min=0.1, max=2.0, step=10, precision=1, default=1.0,
                                              options=set())
+    flare_material = bpy.props.EnumProperty(name="Material",
+                                            description="",
+                                            default=None,
```

Comment:
Is this even legal? In blender 2.80, properties become type annotations, so I'm a little hesitant on this.